lib/fog/compute/google/requests/insert_instance_template.rb

```diff
@@ -0,0 +1,23 @@
+module Fog
+  module Compute
+    class Google
+      class Mock
+        def insert_instance_template(_name, _properties, _description)
+          Fog::Mock.not_implemented
+        end
+      end
+
+      class Real
+        def insert_instance_template(name, properties, description)
```

Comment:
Since it's not entirely clear what `properties` should contain and in what structure, can you please add a doc comment?

E.g.: https://github.com/fog/fog-google/blob/985cd021bcfe4f403deb7c0077997529d529b4b1/lib/fog/compute/google/requests/insert_server.rb#L38-L60